datajoint/erd.py

```diff
@@ -121,6 +121,18 @@ def __init__(self, source, context=None):
                     if node.startswith('`%s`' % database):
                         self.nodes_to_show.add(node)
 
+            for full_table_name in self.nodes._nodes.keys():
+                #one entry per parent
+                parents = connection.dependencies.parents(full_table_name,True)
+                
+                #total count of all the foreign primary keys
+                foreign_primary_keys = sum(len(parent['attr_map']) for parent in parents.values())
+
+                #distinguished table if table introduces atleast on primary key in the schema
+                self.nodes._nodes[full_table_name]['distinguished'] = (True 
+                    if foreign_primary_keys < len(self.nodes._nodes[full_table_name]['primary_key'])
```

Comment:
```
File "/home/travis/build/datajoint/datajoint-python/datajoint/erd.py", line 133, in __init__
    if foreign_primary_keys < len(self.nodes._nodes[full_table_name]['primary_key'])
KeyError: 'primary_key'
```

Reply:
yeah i'm looking into it. it's not giving me that error in my windows vs code. not even in the datajoint docker image.

Reply:
https://github.com/datajoint/datajoint-python/blob/d4254e78f306bbaad865adc0969e09ee9674c788/datajoint/erd.py#L124